grakn-test-tools/src/main/java/ai/grakn/util/EmbeddedCassandra.java

```diff
@@ -69,4 +81,19 @@ public static void start(){
             }
         }
     }
+
+    public static boolean inCassandraContext() {
+        return IN_CASSANDRA_CONTEXT.get() > 0;
+    }
+
+    @Override
+    protected void before() throws Throwable {
+        start();
```

Comment:
If we make `start` private or inline it, we can basically force people to use the context, which is good.

Reply:
Agreed I will do that as part of killing `GraknTestSetup.startCassandraIfNeeded` in another PR though. This one is getting bulky as is.